### ISSUE TITLE
Change in default millennium in from_format in 2.x?

### ISSUE THREAD

**Issue by austinbutler** [sdispater/pendulum#204]
In 2.x I gather that `from_format` now needs to use the former "alternative" formatting, so I changed my code. I was surprised to find that a date previously parsed as 2018 now is parsed as 1918.

Pendulum 1.5.1:
```
>>> pendulum.from_format('02/13/18 15:00', '%m/%d/%y %H:%M')
<Pendulum [2018-02-13T15:00:00+00:00]>
```

Pendulum 2.0.0:
```
>>> pendulum.from_format('02/13/18 15:00', 'MM/DD/YY HH:mm')
DateTime(1918, 2, 13, 15, 0, 0, tzinfo=Timezone('UTC'))
```

Is this intentional? If so, it seems like an odd default.

**Comment by sdispater** [comment]
No, this is not intentional and it's something that slipped through.

I think the right behavior is indeed to use the current century for 2-digits years.

**Comment by sdispater** [comment]
It's fixed in the latest release `2.0.1`
